### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by kunalvasudevan**
can i update print statement

### CODE REVIEW

File: droidbot/adapter/droidbot_app.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def set_up(self):
 
         # device.start_app(droidbot_app)
         while ACCESSIBILITY_SERVICE not in device.get_service_names() and self.__can_wait:
-            print "Please enable accessibility for DroidBot app manually."
+            print("Please enable accessibility for DroidBot app manually.")
             time.sleep(1)
```
Comment (kunalvasudevan) [inline droidbot/adapter/droidbot_app.py:87]: //leave one line

File: droidbot/adapter/droidbot_ime.py [hunk 2]
```diff
@@ -76,7 +76,7 @@ def disconnect(self):
         r_disable = self.device.adb.shell("ime disable %s" % IME_SERVICE)
         if r_disable.endswith("now disabled"):
```
Comment (kunalvasudevan) [inline droidbot/adapter/droidbot_ime.py:77]: if r_disable.endwith("NOW DISABLED"):

**Review by kunalvasudevan — COMMENTED** [review]
updating the code

**Comment by ylimit** [comment]
Close this PR since python 3 has been supported. Thanks!

**Pull request closed without merging**